src/domain/occurrences/OccurrencesPage.tsx

```diff
@@ -49,13 +49,16 @@ const OccurrencesPage: React.FC = () => {
       variables: { id: eventId, include: ['location'] },
     }
   );
+  console.log(eventData);
```

Comment:
Maybe this extra console.log could be removed?

Reply:
yep :)